tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/base.tsx

```diff
@@ -344,3 +344,27 @@ export const FeatureValueInput = (
     />
   );
 };
+
+// ORBSTATION FEATURES
+
+export const FeatureTextInput = (props: FeatureValueProps<string>) => {
+  return (
+    <TextArea
+      height="100px"
+      maxLength={750}
```

Comment:
To answer a question you left in the discord channel, I _think_ you might be able to do:
```
const shortLength: number = 200;
const longLength: number = 750;
export const FeatureTextInput = (props: FeatureValueProps<string>) => {
   ...
   maxLength={longLength}
```

I have only tried testing to see if the intellisense would redline this and not if it actually works though.

Reply:
This worked, thank you.